lib/src/http/response.dart

```diff
@@ -131,23 +131,23 @@ class Response extends BaseResponse {
 /// status, statusText) are available immediately and synchronously. The
 /// response body is available as a stream of bytes.
 class StreamedResponse extends BaseResponse {
-  StreamedHttpBody? _body;
+  late StreamedHttpBody _body;
```

Comment:
can we make this final and pass in value on construction? gets rid need for getter down below and internal setter here

Reply:
I'm not sure how to swing this one. Right now we're relying on the `contentType`, `contentLength`, and `encoding` fields/getters on the response instance itself to create the body from a byte stream, but that only works because the constructor calls `super()` and then the constructor body initializes the field. If we try to do that in a factory constructor, we won't have access to those fields/getters yet, and if we try to do it in a constructor initializer, it has to be before the call to `super()`, so same issue, I think.

Reply:
I see, thanks for explaining the issue!